RockLib.UniversalMemberAccessor.Tests/TypeCreator/Define.cs

```diff
@@ -303,7 +303,7 @@ public static MemberDefinition WriteOnlyProperty(string name, Type type, bool is
             var il = setMethodBuilder.GetILGenerator();
 
             il.Emit(OpCodes.Ldstr, "void " + tb.Name + ".set_" + name + "(" + type.Name + ")");
```

Comment:
Consider changing this to string interpolation

Reply:
Updated to string interpolation.